core/lib/zksync_core/src/state_keeper/batch_executor/main_executor.rs

```diff
@@ -1,4 +1,4 @@
-use std::sync::Arc;
+use std::{fmt::Debug, sync::Arc};
```

Comment:
I imported it because I had to implement `Debug` for some custom type manually in the earlier stages of PR. But good catch that we don't need it anymore.